.style.yapf

```diff
@@ -4,3 +4,5 @@ split_before_expression_after_opening_paren = True
 split_before_first_argument = True
 split_complex_comprehension = True
 split_penalty_comprehension = 2100
+dedent_closing_brackets=True
```

Comment:
@mithro split_before_first_argument was already true from the style.yapf in prjxray. If this is the style should we make it consistent across Symbiflow projects?

Reply:
My suggestions were to add;
```python
DEDENT_CLOSING_BRACKETS=True
INDENT_DICTIONARY_VALUE=True
SPLIT_BEFORE_FIRST_ARGUMENT=True 
```

And you are correct, we should keep them consistent across SymbiFlow repositories. So that looks like;
 * https://github.com/SymbiFlow/symbiflow-arch-defs
 * https://github.com/SymbiFlow/prjxray
 * https://github.com/SymbiFlow/fasm
 * https://github.com/SymbiFlow/fpga-tool-perf
?

Reply:
I'm not sure I understand. Are they case sensitive?

Reply:
No, your version is fine. Are all three values set?

Reply:
Yes